src/Common/src/Interop/Windows/Kernel32/Interop.ReadFile_Bool_SafeHandle_IntPtr.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file is not needed.  The same signature already exists in:
https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/Interop/Windows/Kernel32/Interop.ReadFile_SafeHandle_IntPtr.cs
You just need to update the call site to use `!= 0` on the return value.